packages/ocl/src/Util.js

```diff
@@ -0,0 +1,113 @@
+import { composeNextState } from '@openfn/language-common';
+
+export const Log = (() => {
```

Comment:
Yes please drop this entirely.

Reply:
I am dropping `log` function but I still think we need this available, in all of our adaptors. Because all adaptors should have the same log pattern. Also i think we need a global `Util.js` that can be accessed with any of the adaptor. The file can have generic functions that we don't necessary need them in common, Function like `isObjectEmpty()` or `Log`, 
```
export const log = {
  success: message => console.log(`✓ Success at ${new Date()}: ${message}`),
  warn: message => console.log(`⚠ Warning at ${new Date()}: ${message}`),
  error: message => console.log(`✗ Error at ${new Date()}: ${message}`),
  info: message => console.log(`ℹ Info at ${new Date()}: ${message}`),
};
```

Reply:
_Should_ all our adaptors have the same log pattern? I am not so sure actually.

Timestamps will be added by lightning. If you want timestamps in local, CLI I can add functionality to the logger (so that ALL logs have timestamps). I am not sure of the value of this, and expect this is something you can add to your shell.

If we DO want a standard log pattern in the adaptors, we should be importing `@openfn/logger - because that's a true standard and should give you all the options you need.

Any utility functions used by all adaptors should be exported by common.